editions/tw5.com/tiddlers/howtos/How to export tiddlers.tid

```diff
@@ -0,0 +1,30 @@
+created: 20151006145934884
+modified: 20151006155211988
+tags: Learning
+title: How to export tiddlers
+type: text/vnd.tiddlywiki
+
+!!! Selecting All tiddlers for export
+
+To export ''all'' tiddlers click on the "Tools" tab of the Sidebar. Find and click the ``export all`` ({{$:/core/ui/Buttons/export-tiddler}}) button. A pop-up menu will offer you the chance to export tiddlers in multiple formats. 
+
+!!! Selecting individual tiddlers for export
+
+If you want to export a particular tiddler, first navigate to that tiddler. Then click on the More Actions  icon ({{$:/core/ui/Buttons/more-tiddler-actions}}) that appears above the tiddler. From the list that appears select ``export tiddler``. A pop-up menu will offer you the chance to export the tiddler in multiple formats. 
+
+!!! Selecting tiddlers matching a search criteria for export
```

Comment:
As the tab is called filter, I'd also call that "Exporting tiddlers matching a filter"